lib/rules/selector-max-id/README.md

```diff
@@ -91,7 +91,7 @@ The following patterns are considered problems:
 
 <!-- prettier-ignore -->
 ```css
-a:matches(#foo) {}
+a:is(#foo) {}
```

Comment:
[note] `:matches` was renamed to `:is`. See https://developer.mozilla.org/en-US/docs/Web/CSS/:is